rest_framework/routers.py

```diff
@@ -38,9 +38,7 @@ def escape_curly_brackets(url_path):
     """
     Double brackets in regex of url_path for escape string formatting
     """
-    if ('{' and '}') in url_path:
```

Comment:
`('{' and '}') in url_path` seems strange too, incidentally. This is equivalent to just `'}' in url_path` (I'm guessing it was a typo).

Reply:
Right, I assume this should have been `if '{' in url_path and '}' in url_path:`

I think the intent here was to processes the curly braces as pairs. However, fixing the check still isn't sufficient, as you can still run into weird cases like `} {` or `{{ }` etc.., so it wouldn't necessarily replace matching pairs of braces. 

Removing the check entirely is probably fine.